setup.py

```diff
@@ -9,10 +9,12 @@
     python_requires=">=3.8",
     install_requires=[
         # "watertap @ https://github.com/watertap-org/watertap/archive/main.zip", # uncomment if we need to point to main mid release cycle
+        # "watertap>=1.0.0",
         "watertap==1.0.0rc0",
         "idaes-pse==2.5.0",
         "pyomo==6.7.3",
-        "nrel-pysam == 5.1.0",
+        # "nrel-pysam == 5.1.0",
+        "nrel-pysam == 3.0.2",
```

Comment:
ignore this one, moving forward we will want to be using `nrel-pysam==6.0.0` since that is required for the updated solar thermal models